### PR TITLE
Releasing 5.5.0

### PR DESCRIPTION
**Pull request by carlosroman**

### CODE REVIEW

**Review by iksaif — APPROVED** [review]

**Comment by iksaif** [comment]
The `TestGetTelemetry` test has been flakky for a while :/

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,15 @@
...
 [//]: # (comment: Don't forget to update statsd/telemetry.go:clientVersionTelemetryTag when releasing a new version)
 
-# 5.5.0 / xxxx-xx-xx
+# 5.6.0 / xxxx-xx-xx
 
 
+# 5.5.0 / 2024-01-23
+
+- [FEATURE] Added support for Origin Detection when the container-id is unavailable from inside the application container only with `cgroupv2`. See [#291][].
+- [FEATURE] Added `DistributionSamples` which is similar to `Distribution`, but it lets the client deal with the sampling. The rate is passed to the Agent and not used for further sampling. See [#296][].
```
Comment (maycmlee) [inline CHANGELOG.md:11]:
```suggestion
- [FEATURE] Added `DistributionSamples` which is similar to `Distribution`, but it lets the client handle the sampling. The rate is passed to the Agent and not used for further sampling. See [#296][].
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,9 +2,15 @@
 
 [//]: # (comment: Don't forget to update statsd/telemetry.go:clientVersionTelemetryTag when releasing a new version)
 
-# 5.5.0 / xxxx-xx-xx
+# 5.6.0 / xxxx-xx-xx
 
 
+# 5.5.0 / 2024-01-23
+
+- [FEATURE] Added support for Origin Detection when the container-id is unavailable from inside the application container only with `cgroupv2`. See [#291][].
```
Comment (maycmlee) [inline CHANGELOG.md:10]: Is this correct?

```suggestion
- [FEATURE] Added support for Origin Detection when the container-id is unavailable from inside the application container, only applies to `cgroupv2`. See [#291][].
```

**Review by maycmlee — COMMENTED** [review]
Left a couple of suggestions.

**Pull request merged**